main.py

```diff
@@ -0,0 +1,51 @@
+"""json import"""
+import json
+import os
+
+import mt5_lib as trader
+
+# Path to MetaTrader5 login details.
+ACCOUNT_SETTINGS_PATH = "settings.json"
+
+
+def get_trader_settings(file_path: str) -> dict:
+    """
+    Attempts to deserialize the data in ACCOUNT_SETTINGS_PATH
+    into a json array.
+
+    :param file_path: The file path to deserialize from.
+    :returns dict: A dictionary mirroring the json format.
+    """
+
+    if os.path.exists(file_path):
+        with open(file_path, "r", encoding="utf8") as file:
+            # Convert to json object
+            json_settings = json.load(file)
+            return json_settings
+
+    raise FileExistsError(f"Could not locate resource: {file_path}")
```

Comment:
Next sprint I'll add a test for this. But because I don't have a testing framework set out yet, let's push it in without one

Reply:
Sounds good!